lib/spack/spack/build_environment.py

```diff
@@ -381,7 +394,7 @@ def set_module_variables_for_package(pkg, module):
     m.configure = Executable('./configure')
 
     m.cmake = Executable('cmake')
-    m.ctest = Executable('ctest')
+    m.ctest = MakeExecutable('ctest', jobs)
```

Comment:
We should probably rename `MakeExecutable` to `ParallelExecutable` since most of these aren't `make`. We should also move `ParallelExecutable` to `lib/spack/spack/utils/executable.py`.

Reply:
I can do this.